docs/examples/frameworks/pytorch/pytorch-lightning.ipynb

```diff
@@ -114,32 +114,30 @@
      "output_type": "stream",
```

Comment:
<p>You can also just write (I don't know if we want to bother common user with base_iterator):</p><pre class="ql-syntax" spellcheck="false">from nvidia.dali.plugin.base_iterator import LastBatchPolicy as LastBatchPolicy
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>